feature-libs/product-configurator/rulebased/components/previous-next-buttons/configurator-previous-next-buttons.component.ts

```diff
@@ -68,4 +70,29 @@ export class ConfiguratorPreviousNextButtonsComponent {
       .getNextGroupId(owner)
       .pipe(map((group) => !group));
   }
+
+  protected focusFirstAttribute(): void {
```

Comment:
To make it more readable, can we make it as this:
```
   loading$: Observable<boolean> = this.configRouterExtractorService
    .extractRouterData()
    .pipe(
      switchMap((routerData) =>
        this.configuratorCommonsService.isConfigurationLoading(routerData.owner)
      )
    );

   this.loading$
      .pipe(
        pairwise(),
        filter(([prev, curr]) => prev && !curr),
        take(1),
        delay(0)
      )
      .subscribe(() => this.configUtils.focusFirstAttribute());

```

Reply:
Indeed, there is some repetition that could be avoided. I'm in favour of fixing it later in the RC period. The fix won't break the public API.